<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" idbasedtr="true">
 <author>Bo Phi Yen</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListView" name="lsName">
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="table"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDel">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDup">
          <property name="text">
           <string>Duplicate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionVersion"/>
     <addaction name="actionsss"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <widget class="QMenu" name="mnExport">
     <property name="title">
      <string>Export all text</string>
     </property>
     <addaction name="actionExport_name"/>
     <addaction name="actionExport_to_csv"/>
    </widget>
    <addaction name="mnExport"/>
    <addaction name="actionTranslate_text_file"/>
    <addaction name="actionImport_text"/>
    <addaction name="actionTranslate_auto"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionImport_text">
   <property name="text">
    <string>Import text</string>
   </property>
  </action>
  <action name="actionTranslate_text_file">
   <property name="text">
    <string>Translate text file</string>
   </property>
  </action>
  <action name="actionTranslate_auto">
   <property name="text">
    <string>Translate auto</string>
   </property>
  </action>
  <action name="actionExport_name">
   <property name="text">
    <string>Export name</string>
   </property>
  </action>
  <action name="actionExport_to_csv">
   <property name="text">
    <string>Export to csv</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionsss">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>sss</string>
   </property>
  </action>
 </widget>
 <includes>
  <include location="local">Python Elements.data editor for JD</include>
 </includes>
 <resources/>
 <connections/>
</ui>
